packages/visx-demo/src/sandboxes/visx-xychart/ExampleControls.tsx

```diff
@@ -440,24 +465,35 @@ export default function ExampleControls({ children }: ControlsProps) {
           <label>
             <input
               type="radio"
-              onChange={() => setRenderBarStackOrGroup('bar')}
+              onChange={() => {
+                if (renderAreaLineOrStack === 'areastack') {
+                  setRenderAreaLineOrStack('none');
```

Comment:
previously I disabled updating `renderAreaLineOrStack ` depending on the value of `renderBarStackOrGroup`, I think it's better UX if I just update the value of `renderAreaLineOrStack` so it's valid.